web/src/components/avatar/group-item/index.tsx

```diff
@@ -50,13 +50,13 @@ export const GroupItem: React.FC<GroupItemProps> = props => {
     <FocusRing focusRingClass={onFocus} within>
       <button
         type="button"
-        className={clsx(itemClasses.wrapper, className)}
+        className={clsx(itemClasses.wrapper, className, classes.button)}
         ref={ref}
         aria-label={ariaLabel || name}
         {...buttonProps}
       >
         <Group isUnclickable {...restGroupProps} />
-        <div className={itemClasses.text}>
+        <div className={clsx(itemClasses.text, classes.textWrapper)}>
```

Comment:
Let's keep it consistent, either class name `text`, either `textWrapper`. Should be similar for both I guess